### DISCUSSION TITLE
Stream on the LAN

### DISCUSSION THREAD

**Discussion by Koushikphy** [aminyazdanpanah/python-ffmpeg-video-streaming#112] [General]
Is it possible to create a live streaming link from the webcam feed using this library and open this link on a different machine in the same LAN and play the webcam feed?

**Comment by aminyazdanpanah** [comment]
no matter which network you are using. you can stream your content on LAN, WAN, etc.

**Comment by Koushikphy** [comment]
How do I do that using this library? is there any example code to guide me?

**Reply by aminyazdanpanah** [reply]
Read the documentation:
https://www.hadronepoch.org/op/python/ffmpeg-streaming/start
